logistics_planning_invoicing/wizards/logistics_schedule_existing_account_move_wizard.py

```diff
@@ -0,0 +1,30 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class LogisticsScheduleExistingAccountMove(models.TransientModel):
+    _name = 'logistics.schedule.existing.account.move.wizard'
+    _description = "Logistics Schedule Add Existing Invoice wizard"
+
+    logistics_schedule_id = fields.Many2one('logistics.schedule', readonly=True)
+    product_id = fields.Many2one('product.product', related='logistics_schedule_id.product_id')
+    carrier_id = fields.Many2one('res.partner', related='logistics_schedule_id.carrier_id')
+    invoice_line_id = fields.Many2one('account.move.line', string="Invoice Line")
+    domain_invoice_line_id = fields.Many2many('account.move.line', compute='_compute_domain_invoice_line_id')
+
+    def _compute_domain_invoice_line_id(self):
+        for record in self:
+            product_id = record.logistics_schedule_id._get_invoice_product()
+            if not product_id:
+                raise ValidationError(_("It is necessary to configure the default products of the logistics planning"))
+            record.domain_invoice_line_id = self.env['account.move.line'].search([('move_id.partner_id', '=', record.carrier_id.id), ('product_id', '=', product_id.id)])
```

Comment:
Could you hook the domain part? When done, we'll be able to inject from sales and purchase logistics addons and extra condition regarding to matching analytic account (that comes from order line linked to the logistics schedule).